smriprep/workflows/anatomical.py

```diff
@@ -620,3 +631,54 @@ def _pop(inlist):
     if isinstance(inlist, (list, tuple)):
         return inlist[0]
     return inlist
+
+
+def _aseg_to_three():
```

Comment:
```suggestion
def _aseg_to_three():
    """
    Map FreeSurfer's segmentation onto a brain (3-)tissue segmentation.
    
    This function generates an index of 255+0 labels and maps them into zero (bg),
    1 (GM), 2 (WM), or 3 (CSF). The new values are set accordingly to BIDS-Derivatives.
    Then the index is populated (e.g., label 3 in the original segmentation maps to label
    1 in the output).
```

Reply:
@effigies, what about something like this?

Reply:
Sure. Though `s/accordingly/according/`, and I might say "The `aseg lookup table <...>`__ is available in the FreeSurfer source.".